src/main/java/bellcurvegod/task/Deadline.java

```diff
@@ -0,0 +1,91 @@
+package bellcurvegod.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import bellcurvegod.exception.EmptyDeadlineDescriptionException;
+import bellcurvegod.ui.Ui;
+
+/**
+ * Encapsulates a task with deadline.
+ */
+public class Deadline extends Task {
+    protected LocalDate by;
+
+    /**
+     * Creates a Deadline.
+     *
+     * @param description Description of the Deadline.
+     * @param by          Deadline.
```

Comment:
Perhaps, the parameter descriptions should not be aligned to follow the coding standards (same with the other JavaDoc comments 😄 )